### PR TITLE
License Tweaks

### PR DESCRIPTION
**Pull request by dvaldivia**
Small adjustments to license page
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/License/utils.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const planDetails = [
   {
     id: 0,
     title: "Community",
-    price: "FREE",
+    price: "N/A",
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/License/utils.ts:21]:
```
 {
    id: 6,
    field: "Annual Architecture Review",
    community: "N/A",
    standard: "Yes",
    enterprise: "Yes",
  },
  {
    id: 7,
    field: "Annual Performance Review",
    community: "N/A",
    standard: "Yes",
    enterprise: "Yes",
  },
```
as per pricing page: standard should not have check marks for `Annual Architecture Review`  and `Annual Performance Review` So 

```
  standard: "No",
```

File: portal-ui/src/screens/Console/License/License.tsx [hunk 2]
```diff
@@ -479,60 +479,64 @@ const License = ({ classes, operatorMode }: ILicenseProps) => {
...
-                        </Grid>
-                        <Grid item xs={6}>
                           <Typography
                             variant="button"
                             display="block"
                             gutterBottom
                             className={classes.licenseInfoTitle}
                           >
-                            Subscription Plan
+                            Requester
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/License/License.tsx:510]: as per subnet license 
`Requester` should be `Requestor`

File: portal-ui/src/screens/Console/License/utils.ts (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/utils.ts:21]: nice catch, fixed

File: portal-ui/src/screens/Console/License/License.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/License/License.tsx:510]: done

**Review by prakashsvmx — APPROVED** [review]
lgtm.

**Review by jinapurapu — APPROVED** [review]

**Pull request merged**